en-US/Characters_Codex/03_Creating_a_Character/Ancestries/Ancestries.md

```diff
@@ -24,10 +24,6 @@ Small characters may struggle with heavy weapons, as explained in ["Equipment"](
 \
 Your speed determines how far you can move when traveling (["Adventuring"](#Adventuring_adventuring)) and fighting ([“Combat”](#Combat_combat)).
 
-**Languages**
-\
-By virtue of your ancestry, your character can speak, read, and write certain languages.
```

Comment:
Huh, I never noticed we still had this.

Do we maybe want to add something similar to the _Cultures_ section instead?